.travis.yml

```diff
@@ -9,7 +9,7 @@ env:
     - MAKEFLAGS="-j2"
 
 dist:
-  - trusty
+  - xenial
```

Comment:
Why?

Reply:
xenial can install right gohs dependencies versions from repository (examples/dpi/setup-deps.sh)
Trusty has too old
(Unfortunately, no hyperscan in both, only in newer ubuntu)
There is option to run even more fresh os in Travis - in docker, i am trying it